test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -978,19 +978,13 @@ static public void testMonitorEnterAndExitOnValueType() throws Throwable {
 	}
 
 
-	@Test(enabled=false, priority=2)
-	static public void testSynchMethodsOnValueTypes() throws Throwable {
+	@Test(priority=2)
+	static public void testStaticSynchMethodsOnValueTypes() throws Throwable {
```

Comment:
Please add a comment mentioning that non-static synchronized method is covered by `testValueTypeHasSynchMethods()`

Reply:
I added this, and also some comments for disabled tests in NullRestrictedTypeOptTests.